docs/api/node.md

```diff
@@ -599,7 +604,7 @@ interface NodeInterviewFailedEventArgs {
 
 ### `"ready"`
 
-This is emitted during the interview process when enough information about the node is known that it can safely be used. The node is passed as the single argument to the callback:
+This is emitted when enough information about the node is known that it can safely be used. The node is passed as the single argument to the callback:
```

Comment:
I would make more clear here what 'enough information' menas, at least say what kind of informations are still missing

Reply:
Done. I've also raised https://github.com/zwave-js/node-zwave-js/issues/1800 to give API users a chance to try the CC interview step again without forgetting everything else we know.